### PR TITLE
media-gfx/yafaray: bump to 3.4.1

### PR DESCRIPTION
**Pull request by waebbl**
Also drop 3.3.0

Closes: https://bugs.gentoo.org/717568
Package-Manager: Portage-2.3.99, Repoman-2.3.22
Signed-off-by: Bernd Waibel <waebbl@gmail.com>

### CODE REVIEW

File: media-gfx/yafaray/yafaray-3.4.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,145 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_6 )
```
Comment (juippis) [inline media-gfx/yafaray/yafaray-3.4.1.ebuild:6]: Can you try with 3.7 and 3.8? Looks like they should work simply.

**Comment by waebbl** [comment]
> Looks like it needs to depend on `boost[nls]`.

Nice catch. Although I tested on an almost clean stage 3, this didn't pop up. I'm gonna add it.

> 
> I tested with `PYTHON_SINGLE_TARGET="python3_7"` and it seemed to work.

The current blender-2.79b only has python-3.6 support, and blender.2.8* versions are only available in overlays, so I removed support for 3.7 and 3.8 for now.

File: media-gfx/yafaray/yafaray-3.4.1.ebuild (hunk 1 above)
Comment (waebbl) [reply media-gfx/yafaray/yafaray-3.4.1.ebuild:6]: Getting conflicts with blender-2.79b, the only blender version in the tree.

**Comment by waebbl** [comment]
Alternatively I can try to split off the blender-exporter into a separate ebuild, so the core yafaray can support different python versions. I might even work with python-r1.eclass this way.
What do you think?

**Pull request closed without merging**